site/publish.sh

```diff
@@ -30,7 +30,7 @@ fi
 
 if [[ "${run_stages[*]}" =~ 'sync-plugins' ]]
 then
-    UPDATES_SITE="updates.jenkins.io"
+    UPDATES_SITE="aws.updates.jenkins.io"
```

Comment:
Is this pointing to the virtual machine on AWS or should it be azure.updates.jenkins.io ?
```suggestion
    UPDATES_SITE="azure.updates.jenkins.io"
```

Reply:
It's expected to point to the AWS VM (aka. `pkg.origin.jenkins.io`) to keep it updated.

I've updated the PR body with explanation and link to the issue: does it explain better?

Reply:
@MarkEWaite many thanks for double checking! I'm self-merging as the checks are OK (and to control the update_center2 generation)